### ISSUE TITLE
Generalize Kanto_app in kuksa.val so that it can be used in broader use cases

### ISSUE THREAD

**Issue by bs-jokri** [max-grzanna/Eclipse-Kuksa-Automotive-Edge-Extension#30]
Currently, the PR (https://github.com/eclipse/kuksa.val/pull/284/files) is very specific for one demo case. In order to use it in other cases and make it mergeable to the main repo it needs to be generalized. There are already some comments on things to do. Which need to be done.

There is also the possibility to have a call with the original contributor to clarify open issues

Epic #3

**Comment by cb6569** [comment]
Starting point for improvements are comments in the pull-request.

**Comment by cb6569** [comment]
Dependent on #29.

**Comment by max-grzanna** [comment]
- Implement Console Outputs to debug the application

**Comment by cb6569** [comment]
Pull request was proposed: https://github.com/eclipse/kuksa.val/pull/325
Feedback to solutions for review comments are still open. 
New ticket will be created in case further work is required.
